handlers/handler.go

```diff
@@ -101,6 +101,25 @@ func (env *Env) PostUserHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(reqUser)
 }
 
+// PatchUserHandler updates specific columns of given user
+func (env *Env) DeleteUserHandler(w http.ResponseWriter, r *http.Request) {
+	userId, err := getUserID(r)
+	if err != nil || userId < 0 {
+		errMsg := "Invalid user ID"
+		log.Println(errMsg + ": " + err.Error())
+		http.Error(w, errMsg, http.StatusBadRequest)
+		return
+	}
+	err = env.DB.DeleteUser(userId)
+	if err != nil {
+		errMsg := "Invalid user ID"
```

Comment:
```suggestion
		errMsg := "Failed to delete user"
```

Reply:
I think this should actually just be an internal server error, because MariaDB doesn't throw an error if the row doesn't exist for DELETE. So an error happening would be unexpected.